package/iframe/require.js

```diff
@@ -0,0 +1,961 @@
+/**
```

Comment:
Currently I'm doing:
`cat src/media/js/lib/almond.js package/iframe/logger.js package/iframe/settings.js package/iframe/messages.js package/iframe/activities.js package/iframe/features.js package/iframe/translations.js package/iframe/main.js > package/iframe/require.js`

Following by the `make iframe_package`. If you have pointers about how to integrate that into the build system or want to do it yourself, by all means, please do, that'll free me to do the new buchets stuff, which I don't want to do without this merged first.